tasks/install_agent.json

```
@@ -0,0 +1,50 @@
+{
+  "description": "Install the OpenVox agent package",
+  "private": true,
+  "input_method": "environment",
+  "parameters": {
+    "version": {
+      "description": "The version of openvox-agent to install",
+      "type": "Optional[String]"
```

Do empty strings make sense here? Otherwise you could update it to:
```suggestion
      "type": "Optional[String[1]]"
```

This prohibits empty strings.